src/methods/createObservableMethod.js

```diff
@@ -13,7 +13,8 @@ export default function createObservableMethod (methodName, passContext) {
   }
   const vm = this
 
-  if (!Rx.Observable.prototype.share) {
+  const share = Rx.share || Rx.Observable.prototype.share
+  if (!share) {
```

Comment:
`const share = rxjs.operators.share` if we're using the umd global

Reply:
same as above